databases/business-sync/fragment-shoebox-snippet.js

```diff
@@ -0,0 +1,40 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc),
+  typeFilter: function(doc, oldDoc) {
+    return createBusinessEntityRegex('shoeboxSnippet\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+').test(doc._id);
+  },
+  allowUnknownProperties: true,
+  immutable: true,
+  propertyValidators: {
+    type: {
+      // The type of shoebox snippet (ie, bank transaction, email, photo, digital receipt)
+      type: 'string',
+      required: true,
+      mustNotBeEmpty: true
+    },
+    source: {
+      // The source of the snippet
+      type: 'string',
+      required: true,
+      mustNotBeEmpty: true
+    },
+    kashooEntityType: {
```

Comment:
As I was working through the types with Amrik I came to the exact same conclusion. This really is the junk drawer. Things get thrown in here and we don't know much about them so I'm not sure it is worthwhile trying to store info about the Kashoo entity it gets connected to.

As it gets processed down the pipeline the next document (`banking-reconciliation`, `tax-filing`) might reference this document as the original source but will have all of the information about the "processing" section of the pipeline with the metadata that Dave mentions. I think this document is for the initial portion of the pipeline, "collection" into the shoebox.